modules/end-user-guide/partials/proc_overriding-a-secrets-mount-path-for-a-workspace-component.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container
+
+[id="overriding-a-secrets-mount-path-for-a-workspace-component_{context}"]
+= Overriding a secret's mount path for a workspace component
+
+WARNING: On 
+ifeval::["{project-context}" == "che"]
+{kubernetes} older than v1.13 and
+endif::[]
+OpenShift {ocp3-ver}, secrets mounted as a file override volume mounts defined in the devfile.
+
+You can override a secret's mount path for a workspace component by editing the devfile.
+
+.Prerequisites
+
+* The secret is mounted as a file.
+
+.Procedure
+
+. Declare a component alias for the containers in the override path.
+. Declare an additional volume for the component.
+. Use the name of the secret as the name of the volume.
+. Set the new mount path for the secret as `containerPath`.
++
+.Example: Overriding a secret's mount path for a workspace component
```

Comment:
```suggestion
.Example: Overriding the mount path of a secret for a workspace component
```

Reply:
I'm changing it to *Overriding the mount path of the secret for a workspace component*.